public/stylesheets/login-local.css

```diff
@@ -0,0 +1,32 @@
+body {
+    padding-top: 90px;
```

Comment:
Created a global.css file that modifies the body. Right now it just includes font and padding at the top, but we can add more.